modules/running-compliance-scans.adoc

```diff
@@ -8,6 +8,11 @@
 
 You can run a scan using the Center for Internet Security (CIS) profiles. For convenience, the Compliance Operator creates a `ScanSetting` object with reasonable defaults on startup. This `ScanSetting` object is named `default`.
 
+[NOTE]
+====
+For all-in-one master and worker nodes, compliance scan runs twice on the worker and master nodes. Compliance scan might generate inconsistent scan results. You can avoid inconsistent results by defining only a single role on the master or worker role in the `ScanSetting` object.
```

Comment:
We shouldn't be using "master" any more. Please replace "master" -> "control plane" where you can. I know that the actual role name is still called `master` though, so need to adjust a little there. 

```suggestion
For all-in-one control plane and worker nodes, the compliance scan runs twice on the worker and control plane nodes. The compliance scan might generate inconsistent scan results. You can avoid inconsistent results by defining only a single role in the `ScanSetting` object.
```
The last sentence didn't sound quite right, so I removed "on the master or worker role", because I think it's right without it. But if you do need to mention the role names, then you could do "... by defining only a single `master` or `worker` role in the ..."

Reply:
Addressed